create-llama.sh

```diff
@@ -2,10 +2,7 @@
 
 echo -e "\nAdding sources from create-llama..."
 
-# Remove any existing copy
-rm -rf app/api/chat/config
-rm -rf app/api/files
```

Comment:
@thucpn we still need those two because they come from the copy routes job below, I was just talking about `config`